addons/website/views/res_config_settings_views.xml

```diff
@@ -202,13 +202,22 @@
                                         required="has_plausible_shared_key"/>
                                 </div>
                             </div>
+                            <div invisible="not has_plausible_shared_key">
+                                <button 
+                                    type="action"
+                                    name="%(website.backend_dashboard)d"
+                                    string="See Analytics Report"
+                                    class="btn-link fw-normal"
+                                    icon="oi-arrow-right d-inline"
```

Comment:
```suggestion
                                    icon="oi-arrow-right"
```

After doing this change you will get slight space at left side so to fix this you have to fix below widget.